spec/features/merchants/dashboard_spec.rb

```diff
@@ -25,4 +68,15 @@
       expect(current_path).to eq("/merchants/#{@merchant1.id}/items")
     end
   end
+
+  describe 'items ready to ship' do
+    it 'shows section for items ready to ship with names of items not yet shipped' do
+      expect(page).to have_content("Items Ready to Ship")
+      expect(page).to have_content(@invoice_1.created_at.strftime("%A, %B %d,%Y"))
```

Comment:
Nice, greg and I were wondering how to do the date.

Reply:
Yes, this way is very clean